src/FixedDataTable.js

```diff
@@ -959,7 +999,8 @@ var FixedDataTable = createReactClass({
     }
 
     var lastScrollToRow  = oldState ? oldState.scrollToRow : undefined;
-    if (props.scrollToRow != null && (props.scrollToRow !== lastScrollToRow || viewportHeight !== oldViewportHeight)) {
+    var rowHeightChanged = props.scrollToRow != null ? this._scrollHelper.getRowHeightChanged(props.scrollToRow) : 0
```

Comment:
yes, if we keep it, there will be a problem about "update my rowHeightGetter function, and then grid unexpectedly moves me back up to row 10". But if we not keep it, if we have 2000 rows when expand the last row 2000, then collapse it, the last row will not be positioned at the end, seems some strange. and usually if we need scroll to row, we should control row height correctly.